cmd/mdmctl/apply_dep_profile.go

```diff
@@ -16,15 +18,23 @@ func (cmd *applyCommand) applyDEPProfile(args []string) error {
 	var (
 		flProfilePath = flagset.String("f", "", "filename of DEP profile to apply")
 		flTemplate    = flagset.Bool("template", false, "print a JSON example of a DEP profile")
+		flAnchorFile  = flagset.String("anchor", "", "filename of PEM certificate to add to trusted anchors")
```

Comment:
I doubt that a leaf would work in this scenario. Usually trust requires a full chain.